File: truefalse/truefalse.lua
-- print "Hello, world!"
print("Hello, world!")

-- check if a command line argument was provided
if arg[1] then
    -- convert the argument to lower case
    local arg_lower = string.lower(arg[1])

    -- check if the argument is "true" or "false"
    if arg_lower == "true" then
        print("The argument was true.")
    elseif arg_lower == "false" then
        print("The argument was false.")
    else
        print("The argument was not a recognizable boolean value.")
    end
else
    print("No argument was provided.")
end

File: truefalse/truefalse.py
import sys

print("Hello, world!")

# Check if a command line argument was provided
if len(sys.argv) > 1:
    arg = sys.argv[1].lower()

    if arg == 'true':
        print("The argument was true.")
    elif arg == 'false':
        print("The argument was false.")
    else:
        print("The argument was not a recognizable boolean value.")
else:
    print("No argument was provided.")
